ExposureNotification.App/ExposureNotification.App/Services/ExposureNotificationHandler.cs

```diff
@@ -20,16 +20,17 @@ public class ExposureNotificationHandler : IExposureNotificationHandler
 
 		static readonly HttpClient http = new HttpClient();
 
+		public string UserExplanation
+			=> "We need to make use of the keys to keep you healthy.";
```

Comment:
iOS requires this to prompt the user. I would have liked an info.plist text, but I suppose for localization?